bodhi/tests/server/scripts/test_dequeue_stable.py

```diff
@@ -0,0 +1,41 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+This module contains tests for the bodhi.server.scripts.dequeue_stable module.
+"""
+from click import testing
+
+from bodhi.server import models
+from bodhi.server.scripts import dequeue_stable
+from bodhi.tests.server.base import BaseTestCase
+
+
+class TestDequeueStable(BaseTestCase):
+    """
+    This class contains tests for the dequeue_stable() function.
+    """
+    def test_dequeue_stable(self):
+        """
+        Assert that dequeue_stable moves only the batched updates to stable.
+        """
+        runner = testing.CliRunner()
+
+        update = self.db.query(models.Update).all()[0]
+        update.request = models.UpdateRequest.batched
+        update.status = models.UpdateStatus.testing
+        self.db.commit()
+
+        result = runner.invoke(dequeue_stable.dequeue_stable, [])
+        self.assertEqual(result.exit_code, 0)
+        self.assertEqual(update.request, models.UpdateRequest.stable)
```

Comment:
I would expect that you would need to do another query to refresh the ```update```. Does this line not raise an Exception? Usually sqlalchemy gets real upset if you use an object outside of the transaction it came from. You could/should probably put this line first:

```
update = self.db.query(models.Update).all()[0]
```